apps/opik-frontend/src/components/pages-shared/attachments/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useRef } from "react";
+import { VideoIcon } from "lucide-react";
+
+import { useVideoThumbnail } from "@/hooks/useVideoThumbnail";
+import { useIntersectionObserver } from "@/hooks/useIntersectionObserver";
+
+// ============================================================================
+// Types
+// ============================================================================
```

Comment:
🤏 Nit pick:

No need for these comments, and also for `STYLES`. You could use them right in components - I think we should update `rules` for it